frontend/client/components/Profile/index.tsx

```diff
@@ -106,7 +106,6 @@ class Profile extends React.Component<Props, State> {
 
     return (
       <div className="Profile">
-        {/* TODO: customize details for funders/creators */}
         <HeaderDetails
```

Comment:
unrelated: We should probably do HeaderDetails for Proposals as well

Reply:
https://github.com/grant-project/zcash-grant-system/issues/351